patient/.env.example

```
@@ -2,4 +2,5 @@ SUPABASE_URL="your-supabase-url"
 SUPABASE_ANON_KEY="your-supabase-key"
 GOOGLE_WEB_CLIENT_ID="your-google-web-client-id"
 GOOGLE_IOS_CLIENT_ID="your-google-ios-client-id"
+GEMINI_API_KEY="your-gemini-api-key"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)